### PR TITLE
Fix the quantized table order by float value for easy reading

### PR DESCRIPTION
**Pull request by imsheridan**
This PR is to fix the quantized table order for better reading and understanding in [How to Quantize Neural Networks with TensorFlow](https://www.tensorflow.org/performance/quantization#what_representation_is_used_for_quantized_tensors).
Before fix:
<figure>
  <table>
    <tr><th>Quantized</th><th>Float</th></tr>
    <tr><td>0</td><td>-10.0</td></tr>
    <tr><td>255</td><td>30.0</td></tr>
    <tr><td>128</td><td>10.0</td></tr>
  </table>
</figure>
After fix:
<figure>
  <table>
    <tr><th>Quantized</th><th>Float</th></tr>
    <tr><td>0</td><td>-10.0</td></tr>
    <tr><td>128</td><td>10.0</td></tr>
    <tr><td>255</td><td>30.0</td></tr>
  </table>
</figure>

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee @benoitsteiner: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by imsheridan** [comment]
@suharshs Could you pls take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @suharshs: It has been 14 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Review by suharshs — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @suharshs: It has been 29 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @suharshs: It has been 45 days with no activity and the `awaiting review` label was assigned. Can you please take a look?
